crits_scripts/scripts/get_samples_bucket.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import tarfile
+import time
+from io import BytesIO
+from optparse import OptionParser
+
+from crits.core.mongo_tools import mongo_connector, get_file
+from crits.core.basescript import CRITsBaseScript
+import settings
+
+
+class CRITsScript(CRITsBaseScript):
+    def __init__(self, username=None):
+        self.username = username
+
+    def run(self, argv):
+        parser = OptionParser()
+        parser.add_option('-b', '--bucket-list', action='store', dest='bucket', type='string', help='bucket list name')
+        parser.add_option("-o", "--output-file", action="store", dest="outfile", type="string", help="output archive file (no extension)")
+        (opts, args) = parser.parse_args(argv)
+        samples = mongo_connector(settings.COL_SAMPLES)
+        sample_bucket = samples.find({"bucket_list": opts.bucket})
```

Comment:
I would replace the use of pymongo in the above two lines. Do something like

samples = Sample.objects(bucket_list=opts.bucket)

Rest of comments will assume this is done.